events/wc_pandaria_events.txt

```diff
@@ -78,22 +78,349 @@ narrative_event = {
+			is_incapable = yes
+			is_inaccessible_trigger = yes
+		}
+		# Can only appear after the Naga have spawned in the world
+		has_global_flag = naga_resurfaced_flag
+	}
+	
+	# More likely to appear in Tidehunter courts
+	weight_multiplier = {
+		days = 1
+		modifier = {
+			factor = 1.5
+			religion = neptulon_worship
+		}
+	}
+	
+	immediate = {
+		create_character = {
+			random_traits = yes
+			culture = ankoan
+			dynasty = culture
+			religion = neptulon_worship
+			female = no
+			age = 30
+			flag = ai_flag_refuse_conversion
+		}
+		new_character = {
+			remove_lifestyle_trait_effect = yes
+			random_list = {
+				5 = {
+					add_trait = socializer
+				}
+				5 = {
+					add_trait = socializer
+					add_trait = shrewd
+				}
+				5 = {
+					add_trait = administrator
+				}
+				5 = {
+					add_trait = administrator
+					add_trait = shrewd
+				}
+				10 = {
+					add_trait = schemer
+					add_trait = class_rogue_4
+				}
+				5 = {
+					add_trait = schemer
+					add_trait = shrewd
+					add_trait = class_rogue_4
+				}
+				5 = {
+					add_trait = schemer
+					add_trait = shrewd
+					add_trait = class_rogue_5
+				}
+				10 = {
+					add_trait = mystic
+					add_trait = class_shaman_4
+				}
+				5 = {
+					add_trait = mystic
+					add_trait = shrewd
+					add_trait = class_shaman_4
+				}
+				5 = {
+					add_trait = mystic
+					add_trait = shrewd
+					add_trait = class_shaman_5
+				}
+				10 = {
+					add_trait = sturdy
+					add_trait = class_warrior_4
+				}
+				10 = {
+					add_trait = sturdy
+					add_trait = class_hunter_4
+				}
+				5 = {
+					add_trait = sturdy
+					add_trait = strong
+					add_trait = class_warrior_5
+				}
+				5 = {
+					add_trait = sturdy
+					add_trait = strong
+					add_trait = class_hunter_4
+				}
+				5 = {
+					add_trait = sturdy
+					add_trait = strong
+					add_trait = class_warrior_5
+				}
+				5 = {
+					add_trait = sturdy
+					add_trait = strong
+					add_trait = class_hunter_5
+				}
+			}
+			save_event_target_as = ankoan_courtier
+		}
+	}
+	
+	option = {
+		name = EVTOPTA_WCPND_200
+		custom_tooltip = { text = ankoan_courtier_tooltip }
+	}
+}
+
+### WANDERING ISLE PANDAREN ###
+
+# on_bi_yearly_pulse
+# A Giant Seafaring Turtle Sighted! - initial
+character_event = {
+	id = WCPND.299
+
+	is_triggered_only = yes
+	hide_window = yes
+
+	ai = no
+
+	trigger = {
+		OR = {
+			multiplayer = no
+			is_multiplayer_host_character = yes
+		}
+		NOT = { 
+			has_global_flag = wandering_isle_sighted
+			has_game_rule = {
+				name = mists_of_pandaria
+				value = off
+			}
+		}
+		year = 610	# Roughly 3 years before Mists of Pandaria disappear
+	}
+
+	immediate = {
+		set_global_flag = wandering_isle_sighted
+
+		any_player = {
+			narrative_event = { id = WCPND.300 }
+		}
+	}
+}
+
+# This event unlocks the possibility for random Wandering Isle Pandaren courtiers to spawn in the world
+# A Giant Seafaring Turtle Sighted! - pinger
+narrative_event = {
+	id = WCPND.300
```

Comment:
What about this event spawning lore characters like Chen?

Reply:
There are exceptions like Mojo and Chen that left the Wandering Isle before Azerothians were aware of its existence. This event makes the Wandering Isle known to the world and is loosely based on starting zone lore.